### PR TITLE
Resolve VCSWP-20150 (Update SDKs for new Transcribe Utterance Percl command) 

### PR DESCRIPTION
**Pull request by amahadaya**
Closes VCSWP-20150

### CODE REVIEW

**Review by edgeston — APPROVED** [review]
Ruby SDK looks good to me. Approved!

File: docs/SMSTenDLCCampaign.md [hunk 1]
```diff
@@ -72,7 +72,7 @@ instance = Freeclimb::SMSTenDLCCampaign.new(
   help_message: null,
   reference_id: null,
   mock: null,
-  next_renewal_or_expiration_date: Fri Dec 31 18:00:00 CST 1999
+  next_renewal_or_expiration_date: Fri Dec 31 19:00:00 EST 1999
```
Comment (edgeston) [inline docs/SMSTenDLCCampaign.md:75]: Is there a reason why this was changed?

**Review by edgeston — COMMENTED** [review]
Everything looks good post rebase. Just need one question answered on why something was changed

File: docs/SMSTenDLCCampaign.md (hunk 1 above)
Comment (amahadaya) [reply docs/SMSTenDLCCampaign.md:75]: I think this is because it generates from where I am, so it changes the time from CST to EST

**Pull request merged**